File: src/vel_pkg/scripts/vel_node.py
#!/usr/bin/env python3
# coding=utf-8

import rospy
from geometry_msgs.msg import Twist

if __name__ == "__main__":
    rospy.init_node("vel_node")
    
    vel_pub = rospy.Publisher("cmd_vel",Twist,queue_size=10)
    vel_msg = Twist();
    vel_msg.linear.x = 0
    vel_msg.linear.y = 0
    vel_msg.linear.z = 0
    vel_msg.angular.x = 0
    vel_msg.angular.y = 0
    vel_msg.angular.z = 0.3
    # 频率30
    r = rospy.Rate(30) 
    rospy.loginfo("开始启动……")
    while not rospy.is_shutdown():
        vel_pub.publish(vel_msg)
        r.sleep()
    

File: src/vel_pkg/src/vel_node.cpp
#include<ros/ros.h>
#include<geometry_msgs/Twist.h>

int main(int argc, char **argv) {
    ros::init(argc,argv,"vel_node");
    ros::NodeHandle n;
    ros::Publisher vel_pub = n.advertise<geometry_msgs::Twist>("/cmd_vel",10);
    geometry_msgs::Twist vel_msgs;
    vel_msgs.linear.x = 0.1;
    vel_msgs.linear.y = 0;
    vel_msgs.linear.z = 0;
    vel_msgs.angular.x = 0.1;
    vel_msgs.angular.y = 0;
    vel_msgs.angular.z = 0;
    ros::Rate r(3);
    while (ros::ok())
    {
        vel_pub.publish(vel_msgs);
        r.sleep();
    }
    return 0;
}